pyproject.toml

```diff
@@ -181,24 +181,12 @@ filterwarnings = [
     "ignore:unclosed transport <asyncio.sslproto",
     "ignore:numpy\\.ufunc size changed:RuntimeWarning",
     "ignore:numpy\\.ndarray size changed:RuntimeWarning",
-    "ignore:Importing from numpy:DeprecationWarning:scipy",
-    "ignore:Conversion of the second argument:FutureWarning:scipy",
-    "ignore:Using a non-tuple sequence:FutureWarning:scipy",
-    "ignore:Using or importing the ABCs from 'collections':DeprecationWarning",
     "ignore:Unknown pytest\\.mark\\.mpl_image_compare:pytest.PytestUnknownMarkWarning",
     "ignore:Unknown config option:pytest.PytestConfigWarning",
     "ignore:matplotlibrc text\\.usetex:UserWarning:matplotlib",
-    "ignore:The 'text' argument to find\\(\\)-type methods is deprecated:DeprecationWarning",
-    # Triggered by ProgressBar > ipykernel.iostream
-    "ignore:the imp module is deprecated:DeprecationWarning",
     # Ignore a warning we emit about not supporting the parallel",
     # reading option for now, can be removed once the issue is fixed",
     "ignore:parallel reading does not currently work, so falling back to serial",
-    # Ignore deprecation warning triggered by s3fs (fsspec/s3fs#716)
-    "ignore:'cgi' is deprecated and slated for removal in Python 3.13:DeprecationWarning",
```

Comment:
This would be dealt with by #15401.

Reply:
Oh, right, forgot about that. Can you please rebase #15401. Seems to have conflict now.